README.md

```diff
@@ -12,56 +12,14 @@
+---
 
-Alternatively, you can deploy a copy of the demo stack inside a [play-with-docker (PWD)](https://labs.play-with-docker.com) playground:
-
-- Browse [PWD/?stack=portainer-demo/play-with-docker/docker-stack.yml](http://play-with-docker.com/?stack=https://raw.githubusercontent.com/portainer/portainer-demo/master/play-with-docker/docker-stack.yml)
-- Sign in with your [Docker ID](https://docs.docker.com/docker-id)
-- Follow [these](https://github.com/portainer/portainer-demo/blob/master/play-with-docker/docker-stack.yml#L5-L8) steps.
-
-Unlike the public demo, the playground sessions are deleted after 4 hours. Apart from that, all the settings are the same, including default credentials.
-
-## Getting started
-
-- [Deploy Portainer](https://www.portainer.io/installation/)
-- [Documentation](https://documentation.portainer.io)
-- [Building Portainer](https://documentation.portainer.io/contributing/instructions/)
-
-## Getting help
-
-For FORMAL Support, please purchase a support subscription from here: https://www.portainer.io/products/portainer-business
-
-For community support: You can find more information about Portainer's community support framework policy here: https://www.portainer.io/products/community-edition/customer-success
-
-- Issues: https://github.com/portainer/portainer/issues
-- FAQ: https://documentation.portainer.io
-- Slack (chat): https://portainer.io/slack/
-
-## Reporting bugs and contributing
-
-- Want to report a bug or request a feature? Please open [an issue](https://github.com/portainer/portainer/issues/new).
-- Want to help us build **_portainer_**? Follow our [contribution guidelines](https://documentation.portainer.io/contributing/instructions/) to build it locally and make a pull request. We need all the help we can get!
-
-## Security
-
-- Here at Portainer, we believe in [responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure) of security issues. If you have found a security issue, please report it to <security@portainer.io>.
-
-## Privacy
-
-**To make sure we focus our development effort in the right places we need to know which features get used most often. To give us this information we use [Matomo Analytics](https://matomo.org/), which is hosted in Germany and is fully GDPR compliant.**
-
-When Portainer first starts, you are given the option to DISABLE analytics. If you **don't** choose to disable it, we collect anonymous usage as per [our privacy policy](https://www.portainer.io/documentation/in-app-analytics-and-privacy-policy/). **Please note**, there is no personally identifiable information sent or stored at any time and we only use the data to help us improve Portainer.
-
-## Limitations
-
-Portainer supports "Current - 2 docker versions only. Prior versions may operate, however these are not supported.
-
-## Licensing
+For more information how to build and to update visit the [Portainer-Dogu](https://github.com/cloudogu/portainer) repository.## Licensing
```

Comment:
Here is an editing artefact with the title "Licensing"

Reply:
Thanks, good finding even with your comment I needed some seconds to find the tailing licensing header :)

Reply:
Done